docs/references/events.md

```diff
@@ -61,7 +61,7 @@ This Event indicates that the Workflow Execution has unsuccessfully completed an
 | failure                          | Serialized result of a Workflow failure.                                                                                          |
 | retry_state                      | The reason provided for whether the Task should or shouldn't be retried.                                                          |
 | workflow_task_completed_event_id | The [Run Id](/concepts/what-is-a-run-id) of the [WorkflowTaskCompleted](#workflowtaskcompleted) that the Event was reported with. |
-| new_execution_run_id             | The [Run Id](/concepts/what-is-a-run-id) of the new Workflow started by cron or Retry.                                            |
+| new_execution_run_id             | The [Run Id](/concepts/what-is-a-run-id) of the new Workflow started by Cron or Retry.                                            |
```

Comment:
See discussion about cron on line 53.

Reply:
I did this to get organized. I'm still working on it

Reply:
Ah. I jumped the gun. Back to the starting line with me.

Reply:
okay, all good now